docs/deployment.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deployment and CI/CD Process
+---
+
+# Deployment and CI/CD Process
+
+## Overview
+
+DEV relies on GitHub and Travis to deploy continuously to Heroku. If a Pull Request is merged without the "ci skip" in its title, it will be automatically deployed to production once the build steps complete successfully. The process currently takes about 20 minutes to complete and will need a few additional minutes before the change goes live.
+
+## Travis steps
+
+The following steps can be explored in our [.travis.yml](https://github.com/thepracticaldev/dev.to/blob/master/.travis.yml) and [Procfile](https://github.com/thepracticaldev/dev.to/blob/master/Procfile). Some of the steps will be parallelized in the future.
```

Comment:
maybe a colon at the end of future? since it's introducing the list i mean

Reply:
Updated!